azkaban-exec-server/src/main/bash/azkaban-executor-start.sh

```diff
@@ -50,9 +50,12 @@ if [[ -z "$AZKABAN_OPTS" ]]; then
 fi
 # Set the log4j configuration file
 if [ -f $conf/log4j.properties ]; then
-  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties"
+  AZKABAN_OPTS="$AZKABAN_OPTS -Dlog4j.configuration=file:$conf/log4j.properties -Dlog4j.log.dir=$azkaban_dir/logs"
+else
+  echo "Exit with error: $conf/log4j.properties file must be exist."
```

Comment:
" file doesn't exist"?

Reply:
sorry, I'm not native. fix please

Reply:
No problem.  Could you update your PR with the suggested change?